### PR TITLE
Fatima: School grading system 

### PR DESCRIPTION
**Pull request by FatimaFarah93**
I've created the if statements around each score range to ensure that it prints the correct grade for each one.

### CODE REVIEW

File: src/main/java/session2conditionals/SchoolGradingSystemFatima.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+public class SchoolGradingSystemFatima {
+    
+
+    public static void main(String[] args) {
+        int score = 98;
+
+        if (score < 60) {
+            System.out.println("Your score is F");
+        } else if (score > 60 && score < 70) {
```
Comment (owynowyn) [inline src/main/java/session2conditionals/SchoolGradingSystemFatima.java:11]: Tiny changes to make I think:
The condition score > 60 && score < 70 should be score >= 60 && score < 70 to include scores of 60.

File: src/main/java/session2conditionals/SchoolGradingSystemFatima.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package session2conditionals;
+
+public class SchoolGradingSystemFatima {
+    
+
+    public static void main(String[] args) {
+        int score = 98;
+
```
Comment (owynowyn) [inline src/main/java/session2conditionals/SchoolGradingSystemFatima.java:8]: You may also want to add a small check here to make sure the score is within bounds:

> // Validate the score is within the 0-100 range
>         if (score < 0 || score > 100) {
>             throw new IndexOutOfBoundsException("Invalid Scoring range, Please enter a score between 0 and 100");
>         }

Comment (owynowyn) [reply src/main/java/session2conditionals/SchoolGradingSystemFatima.java:8]: Before you then proceed to do your actual if statement checks

**Review by owynowyn — DISMISSED** [review]
Great work again, couple of tiny changes to make :D

**Review by owynowyn — CHANGES_REQUESTED** [review]
Great work again! Tiny changes to make :D

**Review by FatimaFarah93 — COMMENTED** [review]
I've added in the changes requested:
1. The >= to include 60 into the D category
2. The invalid score message

**Review by dwilliams-gs — APPROVED** [review]
Looks good to me. Nice work @FatimaFarah93 🥳

**Pull request merged**